packages/atomic/src/pages/index.html

```diff
@@ -354,6 +343,10 @@
                   background-color: #44a1da;
                   color: white;
                 }
+
+                atomic-result-link a {
+                  color: yellow !important;
+                }
```

Comment:
Should this be checked in?

Reply:
Oopsie, no :D That plus the above is just a random test I was doing locally.

I need to revert that change